slash/general.py

```diff
@@ -32,6 +32,57 @@ async def github(self, interaction: Interaction):
             embed=embed, view=Link("https://github.com/codeman1o1/TIJK-Bot")
         )
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def avatar(
+        self,
+        interaction: Interaction,
+        user: nextcord.Member = SlashOption(
+            description="The user of which you wish to download their pfp",
+            required=True,
+        ),
+    ):
+        """Get the avatar of you or someone else"""
+        user = user or interaction.user
+
+        avatars_list = []
+
+        def target_avatar_formats(avatar):
+            formats = ["JPEG", "PNG", "WebP"]
+            if avatar.is_animated():
+                formats.append("GIF")
+            return formats
+
+        if not user.avatar and not user.guild_avatar:
+            return await interaction.send(f"**{user}** has no avatar set, at all...")
```

Comment:
Please make this an embed sent ephemeral.

Reply:
done